mantidproject/mantid
# PDCalibration improvements

Add an output workspace for giving the user more information on how the calibration went. In the process, discovered that the calibration was pretty bad so converted from Gauss-Markov to simplex for finding final calibration information. 

**To test:**

Try this before and after merging with powgen data and see that there is a diagnostic workspace and the data lines up better.

```python
LoadEventNexus(Filename='PG3_37861', OutputWorkspace='PG3_37861') #/SNS/PG3/IPTS-2767/nexus/PG3_37659.nxs.h5', OutputWorkspace='PG3_37659')
PDCalibration(SignalWorkspace='PG3_37861', TofBinning='300,-0.0005,16667',
              PeakPositions='2.06,1.2615,1.0758,0.892,0.8186,0.7283,0.6867,0.6307,0.5947,0.5642,0.5441,0.515,0.4996,0.4768,0.4645,0.4205,0.3916,0.3499,0.3257,0.3117',
              PeakWindow=0.02, Tolerance=2, MinGuessedPeakWidth=4, MaxGuessedPeakWidth=14,
              MinimumPeakHeight=5, MaxChiSq=50, TZEROrange='0,10', OutputCalibrationTable='PDCalib', DiagnosticWorkspaces='diag') # DiagnosticWorkspaces doesn't exist on `master`
AlignDetectors(InputWorkspace='PG3_37861', OutputWorkspace='PG3_37861',
               CalibrationWorkspace='PDCalib')
```

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
